src/Console/Commands/CraftableInitializeEnv.php

```diff
@@ -123,16 +111,29 @@ private function getDbSettings(): void
     private function setApplicationName(): void
     {
         if (env('APP_NAME') === 'Laravel') {
-            $this->strReplaceInFile(
-                base_path('.env'),
-                'APP_NAME=Laravel',
-                'APP_NAME="Craftable"'
-            );
-            $this->strReplaceInFile(
-                base_path('.env.example'),
-                'APP_NAME=Laravel',
-                'APP_NAME="Craftable"'
-            );
+            $this->updateEnv('APP_NAME', 'Craftable');
+            $this->updateEnv('APP_NAME', 'Craftable', '.env.example');
+        }
+    }
+
+    /**
+     * Determines if the .env file has default database settings
+     * 
+     * @return boolean
+     */
+    private function isDefaultDatabaseEnv(): bool
+    {
+        if ( 
+            version_compare(app()::VERSION, '5.8.35', '<') && 
+                (env('DB_DATABASE') === 'homestead' && 
+                    env('DB_USERNAME') === 'homestead') ||
+            version_compare(app()::VERSION, '5.8.35', '>=') && 
```

Comment:
the default values has been changed in 5.8.35? Sounds not good practice for me.

Reply:
Yep https://github.com/laravel/laravel/commit/6f3d68f67f3dab0e0d853719696ede8dfd9cc4e1